tests/integration/test_hyper_transformer.py

```diff
@@ -124,14 +124,37 @@ def get_transformed_data():
     ]
     return pd.DataFrame({
         'integer.value': [1, 2, 1, 3, 1, 4, 2, 3],
-        'float.value': [0.1, 0.2, 0.1, np.nan, 0.1, 0.4, np.nan, 0.3],
+        'float.value': [0.1, 0.2, 0.1, 0.2, 0.1, 0.4, 0.2, 0.3],
         'categorical.value': [0.3125, 0.3125, 0.9375, 0.75, 0.3125, 0.75, 0.3125, 0.3125],
-        'bool.value': [0.0, np.nan, 0.0, 1.0, 0.0, 1.0, 1.0, 0.0],
+        'bool.value': [0.0, 0.0, 0.0, 1.0, 0.0, 1.0, 1.0, 0.0],
         'datetime.value': datetimes,
         'names.value': [0.3125, 0.75, 0.75, 0.3125, 0.3125, 0.9375, 0.3125, 0.3125]
     }, index=TEST_DATA_INDEX)
 
 
+def get_reversed_data():
+    datetimes = pd.to_datetime([
+        np.nan,
+        '2010-02-01',
+        '2010-01-01',
+        '2010-01-01',
+        '2010-01-01',
+        '2010-02-01',
+        '2010-01-01',
+        '2010-01-01',
+    ])
+    data = pd.DataFrame({
+        'integer': [1, 2, 1, 3, 1, 4, 2, 3],
+        'float': [0.1, 0.2, 0.1, np.nan, 0.1, 0.4, np.nan, 0.3],
+        'categorical': ['a', 'a', np.nan, 'b', 'a', 'b', 'a', 'a'],
+        'bool': [np.nan, np.nan, np.nan, True, np.nan, True, True, np.nan],
```

Comment:
I think we should probably handle this problem in another issue. I don't think mean values should all be replaced with nans